### PR TITLE
chore(readme): Openjs foundation logo + website link on top of document

### PR DESCRIPTION
**Pull request by AugustinMauroy**
The general idea of this proposal is to add the website link above the readme document.
The logo is there just to look good (it could be reduced in size)

### CODE REVIEW

**Comment by joesepi** [comment]
I like this idea. I wonder if we should use some sort of banner instead.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# The OpenJS Foundation Cross Project Council
+<p align="center">
+  <br />
+  <a href="https://openjsf.org">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg">
+      <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200px">
```
Comment (ljharb) [inline README.md:6]: nit:
```suggestion
      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg" />
      <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200px" />
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    </picture>
+  </a>
+</p>
+
+<h1>Cross Project Council</h1>
```
Comment (ljharb) [inline README.md:11]:
```suggestion
# Cross Project Council
```
this can still use markdown

Comment (AugustinMauroy) [reply README.md:11]: Unless we center it. What do you think?

Comment (ovflowd) [reply README.md:11]: Also not keen on aligning the text, but it feels weird to have the logo floating center and the text on the left. IMO, the text should be on the left.

I know you just ctrlc/ctrlv what I've done on the nodejs.org repo, but there's no need to center the logo here, IMHO.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
-# The OpenJS Foundation Cross Project Council
+<p align="center">
+  <br />
+  <a href="https://openjsf.org">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg" />
+      <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200px" />
```
Comment (ljharb) [inline README.md:6]:
```suggestion
      <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200" />
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
-# The OpenJS Foundation Cross Project Council
+<p align="center">
```
Comment (ljharb) [inline README.md:1]:
```suggestion
<p align="left">
```
it looks odd centered when the rest is left-aligned

Comment (ovflowd) [reply README.md]: We don't need the `<p` statement at all then if it's all left aligned no? We can even switch to pure Markdown I guess...

Comment (AugustinMauroy) [reply README.md]: If we follow this idea we will no longer have the dark mode switch

Comment (ovflowd) [reply README.md]: The image tag can still be HTML, but all the rest can be Markdown.

**Review by ljharb — APPROVED** [review]

**Review by tobie — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Pull request merged**